jcheck/src/main/java/org/openjdk/skara/jcheck/DuplicateIssuesCheck.java

```diff
@@ -74,10 +74,27 @@ private void populateIssuesToHashesMap() throws IOException {
         var issues = new ArrayList<org.openjdk.skara.jcheck.Issue>();
         for (var issue : message.issues()) {
             var hashes = issuesToHashes.get(issue.id());
-            if (hashes != null && hashes.size() > 1) {
-                log.finer("issue: the JBS issue " + issue.toString() + " has been used in multiple commits");
-                var uniqueHashes = new ArrayList<>(new HashSet<>(hashes));
-                issues.add(new DuplicateIssuesIssue(issue, uniqueHashes, metadata));
+            if (hashes != null) {
```

Comment:
You can save the additional process if `hashes.size()` is 1, so I recommend leaving this test as it was before:

```
if (hashes != null && hashes.size() > 1) {
```

Reply:
Good point.